public/index.html

```diff
@@ -15,6 +15,10 @@
       href="https://fonts.googleapis.com/css2?family=Rubik:wght@400;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
```

Comment:
Curious to find out if this to fill/thicken the outlines of icons

Reply:
So this is the stylesheet that allows us to use material UI icons with classnames on <span> tags, you can see an example of one in the footer after our names. This is the line Google gives you to add.